### PR TITLE
feat: team settings sso

### PR DESCRIPTION
**Pull request by neurosnap**
Screenshot 2023-11-28 at 2 54 12 PM
Screenshot 2023-11-28 at 2 55 22 PM

### CODE REVIEW

File: src/auth/organization.ts [hunk 1]
```diff
@@ -111,6 +111,25 @@ export const updateOrganization = authApi.patch<Organization>(
...
+    }),
+  });
+
+  yield* next();
+
+  if (!ctx.json.ok) {
+    return;
+  }
+
+  ctx.loader = { message: "Successfully organization SSO enforcement!" };
```
Comment (joeclayallday) [inline src/auth/organization.ts:130]: Typo here change `Successfully` to Successful`

File: src/ui/pages/settings-team-sso.tsx [hunk 2]
```diff
@@ -1,3 +1,623 @@
...
+    if (!validate(data)) return;
+    dispatch(action);
+  };
+
+  useLoaderSuccess(loader, () => {
+    onSuccess();
+  });
+
+  return (
+    <Box>
```
Comment (joeclayallday) [inline src/ui/pages/settings-team-sso.tsx:104]: Wrap everything in a `<Group>` so the page title shows above the card container & Add the this page title `<h2 className={tokens.type.h2}>Single Sign-On</h2>`

File: src/ui/pages/settings-team-sso.tsx [hunk 3]
```diff
@@ -1,3 +1,623 @@
...
+      <Group>
+        <h3 className={tokens.type.h3}>Edit SSO Configuration</h3>
+
+        <div>
+          <h4 className={tokens.type.h4}>Entity ID</h4>
+          <CopyText text={saml.entity_id} />
+        </div>
+
+        <div>
+          <h4 className={tokens.type.h4}>Sign-in URL</h4>
```
Comment (joeclayallday) [inline src/ui/pages/settings-team-sso.tsx:275]: Capitalize `Sign-in URL` so it says `Sign-In URL`

File: src/ui/pages/settings-team-sso.tsx [hunk 4]
```diff
@@ -1,3 +1,623 @@
...
+                SSO must not be enforced to be removed
+              </Banner>
+            ) : (
+              <Button
+                variant="delete"
+                requireConfirm
+                onClick={onRemove}
+                isLoading={rmLoader.isLoading}
+              >
+                Remove SSO Config
```
Comment (joeclayallday) [inline src/ui/pages/settings-team-sso.tsx:343]: Spell out the full word here, Change `Config` to `Configuration`

**Comment by joeclayallday** [comment]
I made all my design tweaks
Screenshot 2023-11-28 at 4 50 43 PM

**Review by grrowe — APPROVED** [review]

**Review by grrowe — APPROVED** [review]

**Pull request merged**